lib/perl/Genome/Config.pm

```diff
@@ -0,0 +1,158 @@
+package Genome::Config;
+
+use strict;
+use warnings;
+
+use File::Find::Rule qw();
+use Genome::Carp qw(croakf);
+use Genome::ConfigSpec qw();
+use Path::Class qw();
+use YAML::Syck qw();
+
+=item get()
+
+C<get($key)> retrieves the configuration value specified by the input key.  It
+does a hierarchival search for the value,
```

Comment:
`hierarchival` should be `hierarchical`.  This sentence might be clearer with a clause introducing the list, e.g. `...search for the value, which examines the following places:`.